### ISSUE TITLE
Demo site is unavailable

### ISSUE THREAD

**Issue by cschmatzler** [mediacms-io/mediacms#494]
**Describe the issue**
Your domain has no DNS records. As such, the website and demo page are both unreachable.

**To Reproduce**
Steps to reproduce the issue:
1. Go to mediacms.io

**Expected behavior**
Website appears

**Comment by mgogoulos** [comment]
Thanks for raising this, the provider Namecheap has suspended the whole mediacms.io domain because there was a spam incident with a temporary DNS I created months ago pointing to a Hetzner ip. Apparently, after deleting the Hetzner server, someone abused the ip address and the fact that I did not delete the DNS entry. I am now in communication with Namecheap to unblock it. I have to say that to me it's crazy that the whole DNS was suspended due to this. Sure Namecheap sent an email and gave me a few days to reply, but unfortunately that was the time I was AFK on holidays! Very bad coincidense. 

Hope this gets resolved very soon
